socorro/external/es/super_search_fields.py

```diff
@@ -1617,6 +1617,18 @@ def apply_schema_properties(fields, schema):
         "query_type": "string",
         "storage_mapping": {"type": "string"},
     },
+    "missing_symbols": {
+        "data_validation_type": "str",
+        "form_field_choices": [],
+        "has_full_version": False,
+        "in_database_name": "missing_symbols",
+        "is_exposed": True,
+        "is_returned": True,
+        "name": "missing_symbols",
+        "namespace": "processed_crash",
+        "query_type": "string",
+        "storage_mapping": {"analyzer": "semicolon_keywords", "type": "string"},
+    },
```

Comment:
This is identical to `modules_in_stack` value after substituting `"modules_in_stack"` with `"missing_symbols"`